cms/djangoapps/contentstore/utils.py

```diff
@@ -432,6 +435,45 @@ def get_unit_url(course_locator, unit_locator) -> str:
     return unit_url
 
 
+def get_certificates_url(course_locator) -> str:
+    """
+    Gets course authoring microfrontend URL for certificates page view.
+    """
+    unit_url = None
```

Comment:
```suggestion
    certificate_url = None
```
Also fix the following

Reply:
Oops, good catch, thanks!